PGIP/GraphQL/Result/Action.hs

```diff
@@ -0,0 +1,9 @@
+{-# LANGUAGE CPP, TypeFamilies, DeriveDataTypeable #-}
+
+module PGIP.GraphQL.Result.Action where
+
+import Data.Data
+
+data Action = Action { evaluationState :: String
+                     , message :: Maybe String
+                     } deriving (Show, Typeable, Data)
```

Comment:
why is this datatype called `Action`? It seems to be more a datatype for storing results of some action?

Reply:
This might be extended to the action datatype that Tim described in his master's thesis. It can get an ETA and more information about the action in the future.

Reply:
good, then please add a corresponding comment.